src/codemagic/cli/cli_help_formatter.py

```diff
@@ -1,11 +1,15 @@
 import argparse
+import re
 import shutil
 import sys
+from typing import Set
 
 from .colors import Colors
 
 
 class CliHelpFormatter(argparse.HelpFormatter):
+    _suppressed_actions: Set[str] = set()
```

Comment:
```suggestion
    _deprecated_actions: Set[str] = set()
```

Reply:
Addressed in [a821077](https://github.com/codemagic-ci-cd/cli-tools/pull/187/commits/a821077e1e61d59a4e4c763bdedacb8e0a4c4ce9).